stdlib/format.mli

```diff
@@ -288,6 +288,44 @@ val print_break : int -> int -> unit
   the current indentation.
 *)
 
+val pp_print_custom_break :
+  formatter ->
+  no_break:(string * int * string) ->
+  yes_break:(string * int * string) ->
+  unit
+(** [pp_print_custom_break ppf ~no_break:(s1, n, s2) ~yes_break:(s3, m, s4)]
+   emits a custom break hint: the pretty-printer may split the line at this
+   point.
+
+   If it does not split the line, then the [s1] is emitted, then [n] spaces,
+   then [s2].
```

Comment:
Do others think that it would be helpful to explicitly note that `out_spaces` is used to emit the spaces, to head off user confusion about why `(s1, n, s2)` isn't just one string (with some space characters in the middle)?